src/main/java/controller/CarRaceController.java

```diff
@@ -49,8 +49,14 @@ private int requestCarMoveCount() {
     private void move(final Map<String, Integer> initialCarStatus, final int moveCount) {
         outputView.printInitialStatus(initialCarStatus);
         for (int count = 0; count < moveCount; count++) {
-            outputView.printMoveResult(carRaceService.move());
+            outputView.printMoveResult(racingGame.playTurn());
         }
     }
 
+    private RacingGame createNewGame(final String cars) {
+        return new RacingGame(
+            Arrays.stream(cars.split(CAR_NAME_DELIMITER)).collect(Collectors.toList()),
```

Comment:
```suggestion
            Arrays.stream(cars.split(CAR_NAME_DELIMITER))
                 .collect(Collectors.toList()),
```
스트림은 연산자마다 줄바꿈을 쓰는 것이 가독성이 좋습니다!